lib/node_modules/@stdlib/_tools/scaffold/math-strided-real-typed-unary/data/benchmark/benchmark__js.txt

```diff
@@ -16,12 +16,14 @@
 * limitations under the License.
 */
 
+/* THIS IS A GENERATED FILE. DO NOT EDIT DIRECTLY. */
+
 'use strict';
 
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var uniform = require( '@stdlib/random/base/uniform' ).factory;
+var random = require( '@stdlib/random/base/random' ).factory;
```

Comment:
@gunjjoshi Same comment. Use `array/zeros` and `random/array/*` to create `y` and `x`, respectively, below.